test/integration/infrastructure/infrastructure_test.go

```diff
@@ -520,12 +546,19 @@ func runTest(
 		}
 
 		By("verify infrastructure creation after migration")
-		verifyCreation(ctx, project, computeService, iamService, infra, providerConfig)
+		verifyCreation(ctx, project, computeService, iamService, infra, providerConfig, dualStack)
 	}
 
 	return err
 }
 
+func verifyDualStackSetup(ctx context.Context, project string, computeService *computev1.Service, namespace string) {
+	// Verify that the subnets have IPv6 CIDR ranges, firewalls allow IPv6 traffic, etc.
```

Comment:
The comment indicates you want to do more checks. For now it just checks IPv6 range of the nodes subnet.
Will you add additional checks?

Reply:
Done